src/code/dialogs/markerAddDialog.js

```diff
@@ -85,12 +85,18 @@ const MarkerAddDialog = WDialog.extend({
     const content = L.DomUtil.create("div", "content");
     this._portal = L.DomUtil.create("div", "portal", content);
 
+    L.DomUtil.create("label", null, content).textContent = "Type";
```

Comment:
todo wX strings

Reply:
Right, I'll do it and add missing translations in the dialog